databuilder/extractor/dashboard/tableau/tableau_dashboard_query_extractor.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from typing import Any, Dict, Iterator
+
+from pyhocon import ConfigFactory, ConfigTree
+
+import databuilder.extractor.dashboard.tableau.tableau_dashboard_constants as const
+from databuilder import Scoped
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.extractor.dashboard.tableau.tableau_dashboard_utils import TableauGraphQLApiExtractor,\
+    TableauDashboardUtils
+from databuilder.extractor.restapi.rest_api_extractor import STATIC_RECORD_DICT
+from databuilder.transformer.base_transformer import ChainedTransformer
+from databuilder.transformer.dict_to_model import DictToModel, MODEL_CLASS
+
+LOGGER = logging.getLogger(__name__)
+
+
+class TableauGraphQLApiQueryExtractor(TableauGraphQLApiExtractor):
+    """
+    Implements the extraction-time logic for parsing the GraphQL result and transforming into a dict
+    that fills the DashboardQuery model. Allows workbooks to be exlcuded based on their project.
+    """
+
+    CLUSTER = const.CLUSTER
+    EXCLUDED_PROJECTS = const.EXCLUDED_PROJECTS
+
+    def execute(self) -> Iterator[Dict[str, Any]]:
+        response = self.execute_query()
+
+        for query in response['customSQLTables']:
```

Comment:
a comment on which API it is using?

Reply:
Could you clarify what you'd like to see? There is a comment for this class that calls out what model is expected (`DashboardQuery`). The API relates to the query found in the [TableauDashboardQueryExtractor class](https://github.com/amundsen-io/amundsendatabuilder/pull/333/files#diff-f5199fa048baa7a5e23e4e8aa94ea53fR66) in this file.

Reply:
I see, I miss that part.  No worry then.